packages/rspack/tests/IgnorePlugin.test.ts

```diff
@@ -0,0 +1,109 @@
+// @ts-nocheck
+import { Compiler, getNormalizedRspackOptions, rspack } from "../src";
+const path = require("path");
+const { IgnorePlugin } = require("webpack");
+
+class Plugin implements RspackPluginInstance {
+	name = "a";
+
+	apply(compiler: Compiler) {
+		// Wait for configuration preset plugions to apply all configure webpack defaults
+		compiler.hooks.compilation.tap("a", com => {
+			com.normalModuleFactory.hooks.beforeResolve.tap("a", (...args) => {
+				console.log(...args);
```

Comment:
console log in test is meaningless, if you want to  check something please use jest.expect or something similar.